app/events.py

```diff
@@ -101,9 +101,10 @@ def getGcalEventObj(event):
             if mapping:
                 placeName = mapping['name']
                 location = mapping['location']
-                yelpId = search._guessYelpId(placeName, location['lat'], location['lng'])
-                if (yelpId):
-                    eventObj = representation.eventRecord(yelpId, location['lat'], location['lng'], summary, event['start']['dateTime'], event['end']['dateTime'], event['htmlLink'])
+                yelpBiz = search._guessYelpBiz(placeName, location['lat'], location['lng'])
+                if (yelpBiz):
```

Comment:
Do we check if the venue is already in firebase? I'm a bit concerned we'll be burning up yelp search queries on venues we know about already.